bundles/io/org.eclipse.smarthome.io.net.test/pom.xml

```diff
@@ -15,17 +15,12 @@
 
   <name>Eclipse SmartHome Network I/O Tests</name>
 
-  <properties>
-    <bundle.symbolicName>org.eclipse.smarthome.io.net.test</bundle.symbolicName>
-    <bundle.namespace>org.eclipse.smarthome.io.net.test</bundle.namespace>
-  </properties>
-
   <build>
     <plugins>
       <plugin>
         <groupId>${tycho-groupid}</groupId>
         <artifactId>target-platform-configuration</artifactId>
-        <configuration>
+        <configuration>    
```

Comment:
A few spaces too much at the end of this line.

Reply:
I let Eclipse format the complete file

Reply:
Is the Eclipse configuration now broken, because it replaced spaces by tabs for this xml file :(

Reply:
Okay, resolved that as well, but we should fix the eclipse settings, maybe its because I (accidentally) upgraded to photon :-)